zubhub_frontend/zubhub/src/views/activities/activities.jsx

```diff
@@ -0,0 +1,37 @@
+import React,{useEffect} from 'react'
+import {connect} from 'react-redux'
+import {getActivities} from '../../store/actions/activityActions'
+
+
+function Activities(props) {
+   const activities = props.activities.all_activities
+   useEffect(() => {
+     props.getActivities()
+   }, [])
+   
+  return (<div>
+             <h1>activities</h1>
+             {activities? 
+               activities.map(activity => (
+                <h2>{activity.title}</h2>
+               ))
+              :'No activities created yet!!'}
```

Comment:
Missing translation?

Reply:
Thanks for mentioning that I will make sure to add it, for this PR the task is just to connect the store to this component